Frontend/src/Components/Home/Home.jsx

```diff
@@ -14,11 +14,6 @@ const Home = () => {
     }
   };
 
-  const handleLogout = () => {
```

Comment:
why did you remove this?

Reply:
I moved the function to my header because thats where my logout button is